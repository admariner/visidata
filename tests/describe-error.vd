sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/test.jsonl	o	
test	key1		type-date		@	
test	qty		type-int		#	
test	amt		type-float		%	
test			describe-sheet		I	
test_describe	column		sort-asc		[	
test_describe	mode		hide-col		-	hide current column
test_describe	sum		type-float		%	set type of current column to float
