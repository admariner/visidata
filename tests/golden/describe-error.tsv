column	errors	nulls	distinct	min	max	sum	median	mean	stdev
	0	10	2						
amt	0	3	8	0.01	43.2	58.92	3.0	8.42	15.67
key1	0	3	9	2016-01-01	2018-10-20	#ERR	2018-07-27	#ERR	#ERR
key2	0	3	5						
qty	1	2	9	1	256	375.00	16	53.57	92.05
